### PR TITLE
Allow building with preinstalled openhome libs

### PR DESCRIPTION
**Pull request by joerg-krause**
Adding the config options `--with-ohnet=`, `--with-ohnetgenerated=`, and `--with-ohsongcast=` to allow building with preinstalled static libraries of the openhome family.

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -67,6 +86,10 @@ sc2mpd_SOURCES = \
      sc2src/wav.h \
      sc2src/workqueue.h
 
+if WITH_OHBUILD
```
Comment (medoc92) [inline Makefile.am]: Why? Except if I'm mistaken, this just excludes ohbuild.sh from the 'make dist' which does not seem like a good idea?

Comment (joerg-krause) [inline Makefile.am]: My bad! I am not that familiar with autotools. So, I'll remove this.

**Comment by joerg-krause** [comment]
I've updated the PR.

**Comment by medoc92** [comment]
I've found a more correct way to include ohbuild.sh, just pushed an update.

I'd be interested by how you build the openhome libs if you have a pointer to this. The source I use is 2 years old, it may be time that I update.

I am going to release an 1.1.2 with the recent changes.

**Comment by joerg-krause** [comment]
Thanks for the update!

I'm using Buildroot to build the openhome libs. For now, I' using my [oh](https://github.com/joerg-krause/buildroot/tree/oh) branch, but I am planning to get it into mainline. Just trying to minimize the dependencies.

Maybe you can wait for the release some days, so I can verify everything is working? At least, sc2mpd can be compiled for almost all architectures available in Buildroot :smile:

**Comment by medoc92** [comment]
Yes, no hurry obviously.

**Pull request merged**